### ISSUE TITLE
filter dont work on last version

### ISSUE THREAD

**Issue by Menogus** [aslagle/reactive-table#194]
on 0.7.2  dont work on node.js (0.12.0) server (on meteor - all ok)
on 0.6.15 - all ok

**Comment by aslagle** [comment]
I tried one of my examples on node 0.12.2 and it worked fine. Are you getting any error messages?

**Comment by aminjam** [comment]
I ran into the same issue. Filter stops working when there is a `sort: 'descending'` on a column

**Comment by aslagle** [comment]
I still can't reproduce this, on node 0.12.2 with `sort: 'descending'`.

**Comment by hitchcott** [comment]
+1 - I'm having the same issue, but it fixes itself as soon as I start sort any of the columns: Repro for me:

1. Template containing table loads, table has data
2. Start typing the filter box, nothing happens, press enter, nothing happens
3. Click any of the column headers to sort -- the filter is now applied and working
4. From now on, typing in the filter box listens to keyup as expected

So initiating a sort seems to fix the issue (but not if using a `sort` option).

This is on a project I just updated from 0.8 to 1.2, it was working before & nothing else has changed.

Table settings looks like this:

```
    rowsPerPage: 30
    useFontAwesome: true
    showNavigation: 'always'
    fields:[
        key: 'profile.last_name'
        label: TAPi18n.__ 'last_name'
      ,
        key: 'profile.first_name'
        label: TAPi18n.__ 'first_name'
      ,
        key: 'profile.company'
        label: TAPi18n.__ 'company'
      ,
        key: 'sign_ins'
        label: TAPi18n.__ 'signed_in'
        fn: (val,obj) -> if val && val.indexOf(sessionId) > -1 then '✓'
    ]
```

Will look into it...

**Comment by Menogus** [comment]
yep...i still use 0.6.15 version of reactiveTable :)) with nodejs 4.2.1 now

**Comment by sirpy** [comment]
this happened to me, but adding {{#if Template.subscriptionsReady}} solved it

**Comment by aslagle** [comment]
If anyone was still having this problem, I think it's finally fixed. Try version 0.8.29!
